dist/pom.xml

```diff
@@ -78,7 +78,7 @@
                                             <fileset dir="../driver/target" includes="*.jar" />
                                         </copy>
                                         <copy todir="target/appassembler/repo/cluster">
-                                            <fileset dir="../driver/target/cluster" includes="**/*.jar" />
+                                            <fileset dir="../driver/target/cluster" includes="**/*.jar" excludes="**/input-rabbitMQ*.jar" />
```

Comment:
running `find . -name "*rabbit*jar"` in an installation returns:

./plugins/original-input-rabbitMQ-plugin.jar
./repo/plugins/original-input-rabbitMQ-plugin.jar